kzz7yh-g04217/cod-ve09v3_kzz7yh-g04217.xml

```diff
@@ -74,13 +74,14 @@
             <p xml:id="kzz7yh-g04217-d1e127">
       ¶ 
       Quar<lb ed="#V" n="102" break="no"/>to vtrum simul tempore fuerit vnio carnis et anime et 
-      assun<lb ed="#V" n="103" break="no"/>ptio vtriusque ad vnitatem diuine persone. 
+      assum<lb ed="#V" n="103" break="no"/>ptio vtriusque ad vnitatem diuine persone. 
```

Comment:
I don't think the locations of the line breaks are correct. From how I see it, the lines should be "quarto queritur vtrum simul tempore fue / rit carnis conceptio et vnio carnis et anime: et / assumptio ..."